locust_plugins/csvreader.py

```diff
@@ -24,13 +24,15 @@ def __next__(self):
 class CSVDictReader:
     "Read test data from csv file using an iterator"
 
-    def __init__(self, file):
+    def __init__(self, file, fieldnames=None, restkey=None, restval=None, dialect="excel", *args, **kwds):
```

Comment:
Same here. If you want to change the defaults then maybe you should have a subclass of CSVDictReader in your own code.